### ISSUE TITLE
Replace Ascii art with custom image

### ISSUE THREAD

**Issue by o2sh** [o2sh/onefetch#104]
It's quite a long shot, but here we go...

It would be nice if a user could choose to display an image (*png, *jpg) instead of the ascii art.

This would be done via flag, as such:

`onefetch --source ~/pictures/my-picture.png`

![tempsnip](https://user-images.githubusercontent.com/13710835/67337365-2e403400-f527-11e9-94d8-0c4b28525114.png)

For the brave only 💯 🥇

**Comment by andymac-2** [comment]
If you know that your terminal supports X11 you should be able to do this. For a more cross platform solution you'll probably need to open a popup window using something like SDL

**Comment by CephalonRho** [comment]
Hey, I made a pull request to add support for this with the kitty terminal graphics protocol ( #113 ).
This works pretty well, but has very limited support (right now only kitty itself implements that protocol) so I also added a basic trait to have multiple image backends for different terminal emulators.
Other options for displaying images like this are Sixel or more emulator specific escape codes.
Using an overlay to display it over any terminal emulator might work if the cell size and window position can be determined, but it would require making onefetch interactive or spawning a background process and tracking the images position in the terminal somehow.
